src/main/kotlin/org/oso/core/entities/VerificationToken.kt

```diff
@@ -0,0 +1,22 @@
+package org.oso.core.entities
+
+import java.time.Instant
+import java.util.*
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.OneToOne
+
+@Entity
+data class VerificationToken(
+    @Id
+    val token: String,
+    @OneToOne
+    val helpRequester: HelpRequester,
+    @OneToOne
+    val helpProvider: HelpProvider,
+    val expiryDate: Instant = Calendar.getInstance().apply { add(Calendar.MINUTE, EXPIRATION) }.toInstant()
```

Comment:
we used LocalDateTime for times until now, we should stick to one time-library

Reply:
You are right.